client/src/components/ProfileSettings/EditProfile/EditProfileForm/useStyles.ts

```diff
@@ -0,0 +1,64 @@
+import { makeStyles, Theme } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  form: {
+    width: '100%',
+    height: '100%',
+    alignItems: 'center',
+  },
+  toggle: {
+    marginBottom: '1%',
+  },
+  gridItemA: {
+    display: 'flex',
+    flexFlow: 'row wrap',
+    justifyContent: 'flex-start',
+    alignContent: 'center',
+    alignItems: 'center',
+    fontWeight: 'bold',
```

Comment:
You might not need most of these as they are defined as props you can pass to the `Grid` component (listed [here](https://material-ui.com/api/grid/#props)).